src/app/src/test/kotlin/org/imperial/mrc/hint/database/ProjectRepositoryTests.kt

```diff
@@ -60,6 +60,21 @@ class ProjectRepositoryTests
         assertThat(project.versions[0].updated).isNotNull()
     }
 
+    @Test
+    fun `can get project notes`()
+    {
+        val uid = setupUser()
+
+        val projectId = sut.saveNewProject(uid, "testProjectRepo", note = "notes")
+        versionRepo.saveVersion("v1", projectId, "notes")
```

Comment:
I would something other than `"notes"` on this line - just to ensure that the you're not getting the project note returned for the version.